src/simplepolyhedron.jl

```diff
@@ -75,6 +75,11 @@ function Base.push!(p::SimplePolyhedron{N}, ext::VRepresentation{N}) where N
     p.hrep = nothing
 end
 
+"""
+    hrepiscomputed(p::SimplePolyhedron)
```

Comment:
The argument should be `Polyhedron`. Can you also move the doc to the definition in `operations.jl` ?